.github/workflows/e2e_tests.yml

```diff
@@ -17,6 +17,8 @@ jobs:
   execute:
     name: Start tests
     runs-on: ubuntu-latest
+    permissions:
+      contents: none
```

Comment:
📣 Permissions are handled within CircleCI for this workflow:

https://github.com/slackapi/slack-cli/runs/40890474475